controllers/set/infrastructure_status.go

```diff
@@ -138,3 +142,14 @@ func extractInfrastructureNameFromSeedNamespace(namespace string) (string, bool)
 
 	return strings.Join(parts[2:], "--"), true
 }
+
+func sortUntypedStringSlice(s []any) {
```

Comment:
uh, there is a helper in netip for this !

Reply:
Generic sort in netip? :sweat_smile:

Reply:
I was able to get rid off the untyped slice, so now a simple slice sort is sufficient to do the job.